### ISSUE TITLE
Cannot pass number value to loop prop

### ISSUE THREAD

**Issue by superdumb93** [Gamote/lottie-react#30]
**Describe the bug**
I get this typescript error when I pass a number value to loop prop of Lottie component.
```
Type 'number | boolean | undefined' is not assignable to type 'boolean | undefined'.
  Type 'number' is not assignable to type 'boolean | undefined'.
```

**To Reproduce**
`<Lottie loop={3} />`

**Expected behavior**
Error should not occur.

**Screenshots**
None

**Comment by xiaohong14** [comment]
The reason was related with this line in the codebase.
`export type LottieComponentProps = LottieOptions &
  React.HTMLProps<HTMLDivElement> & {
    interactivity?: Omit<InteractivityProps, "lottieObj">;
  };`

React.HTMLProps already has the same loop prop, so it was replacing the LottieOptions['loop'] prop.

**Comment by yoanad** [comment]
Is there any workaround for that?

**Comment by Gamote** [comment]
This issue was fixed in [`v2.3.1`](https://github.com/Gamote/lottie-react/releases/tag/v2.3.1) by @mattvague.
